src/main/java/uk/gov/ons/census/casesvc/service/CaseProcessor.java

```diff
@@ -16,13 +16,14 @@
 public class CaseProcessor {
 
   private static final String SURVEY = "CENSUS";
+  private static final String CASE_UPDATE_ROUTING_KEY = "event.case.update";
```

Comment:
would this make more sense as we're emitting a CaseCreated event here?
```suggestion
  private static final String CASE_CREATE_ROUTING_KEY = "event.case.create";
```

Reply:
this is what's in the spec here: https://collaborate2.ons.gov.uk/confluence/display/CATD/RM+Event+Model

Has been agreed with Rob Smart, Phil Whiles & Ella Cook... so gonna mark this as resolved.